server/src/notification.js

```diff
@@ -0,0 +1,24 @@
+import fetch from 'node-fetch';
+
+const FIREBASE_SERVER_KEY = "AIzaSyBNjq8QTFCVLRYs9EyXa5UZjmJdSEPFxjA";
+
+const FIREBASE_ROOT_URL = 'https://fcm.googleapis.com/fcm';
+
+export const sendNotification = (notification) => {
+  fetch(`${FIREBASE_ROOT_URL}/send`, {
```

Comment:
this API looks like it supports promises so just add `return` before `fetch` to have it return the promise to be awaited by the caller.

Reply:
(Y)